UniversityDatabase/Database.cpp

```diff
@@ -1,22 +1,76 @@
 #include "Database.h"
 
-void Database::AddPerson(Person newPerson)
+void Database::AddPerson(const Person newPerson)
 {
 	Persons.push_back(newPerson);
 }
 
-std::vector<Person> Database::GetPersons()
+People Database::GetPersons()
 {
 	return Persons;
 }
 
-Person Database::SearchByName(std::string Name)
+void Database::RemovePersonByPesel(const std::string pesel)
 {
-	for (std::vector<Person>::iterator it = Persons.begin(); it < Persons.end(); it++)
+	for (People::iterator it = Persons.begin(); it < Persons.end(); it++)
 	{
-		if (it->CheckName(Name))
+		if (pesel == it->GetPesel())
+		{
+			RemovePerson(it);
+			break;
+		}
+	}
+}
+
+void Database::RemovePerson(People::iterator it)
+{
+	Persons.erase(it);
+}
+
+People Database::SearchBySurname(const std::string surname)
+{
+	People foundPeople;
+	for (People::iterator it = Persons.begin(); it < Persons.end(); it++)
+	{
+		if (surname == it->GetSurname())
+		{
+			foundPeople.push_back(*it);
+		}
+	}
+	return foundPeople;
+}
+
+Person Database::SearchByPesel(const std::string pesel)
+{
+	for (People::iterator it = Persons.begin(); it < Persons.end(); it++)
+	{
+		if (pesel == it->GetPesel())
 		{
 			return *it;
 		}
 	}
+}
+
+void Database::PrintPersonalInformation(Person person)
+{
+	std::cout << person.GetName() << std::endl;
+	std::cout << person.GetSurname() << std::endl;
+	std::cout << person.GetPesel() << std::endl;
+	std::cout << person.GetAddress() << std::endl << std::endl;
+}
+
+void Database::PrintPeople(People people)
+{
+	for (People::iterator it = people.begin(); it < people.end(); it++)
```

Comment:
It is good practice to use a Range-based loop. Try to implement it where you iterate over an object in a vector.
https://www.geeksforgeeks.org/range-based-loop-c/